xc/common/utils/prjxray_create_ioplace.py

```diff
@@ -128,6 +145,21 @@ def main():
             sys.exit(1)
 
         loc, is_output, iob = pad_map[pcf_constraint.pad]
+
+        if args.synth_tiles:
+            tile = get_synth_tile_from_pad(synth_tiles, pcf_constraint.pad)
+            if tile:
```

Comment:
When would `get_synth_tile_from_pad` return None, and it not be an error?

Reply:
In the case of overlays, where the architecture has both synth IOs and real IOs (so not all pcf_constraint pads would be in the synth_tiles).

Reply:
Ah, sure.  Please add a comment about that, that is not obvious, but does make sense in retrospect.  Should there be logic to make sure that it isn't possible to have an ambiguous match between the synth tile and the "real" IO tiles?